modules/ob-release-notes-1-2.adoc

```diff
@@ -0,0 +1,42 @@
+// This module is included in the following assemblies:
+// * about/ob-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-release-notes-1-1_{context}"]
```

Comment:
```suggestion
[id="ob-release-notes-1-2_{context}"]
```
This file needs changing everything from 1.1 to 1.2. I know you showed me those changes so maybe just the file did not get saved or something :smile: